spatial-tracking-explainer.md

```diff
@@ -92,7 +92,7 @@ function onSessionStarted(session) {
 There is no mechanism for getting a floor-relative _unbounded_ reference space. This is because the user may move through a variety of elevations (via stairs, hills, etc), making identification of a single floor plane impossible.
 
 ### Stationary experiences
-A _stationary_ experience is one which does not require the user to move around in space.  This includes several categories of experiences that developers are commonly building today, and multiple reference space types are available to address each class of experience.  "Standing" experiences can be created with a `floor-level` reference space.  "Seated" experiences can be created with an `eye-level` reference space.
+A _stationary_ experience is one which does not require the user to move around in space.  This includes several categories of experiences that developers are commonly building today, and multiple reference space types are available to address each class of experience.  "Standing" experiences can be created with a `floor-level` reference space.  "Seated" experiences can be created with an `eye-level` reference space. Orientation-only experiences such as 360 photo/video viewers can also be created with an `eye-level` reference space, though in that case positional data should be ignored in order to display the media properly.
```

Comment:
>though in that case positional data should be ignored in order to display the media properly.

Is this actually the guidance?  Thought it was to render the content on sufficiently far away surrounding geometry....

Reply:
Either approach works, and there's minor pros/cons to each but no so much that it really makes a difference in terms of guidance here.

Reply:
Then let's say something like this?

> Orientation-only experiences such as 360 photo/video viewers can also be created with an `eye-level` reference space, and in that case one method would be to ignore the positional data in order to display the media properly.

Reply:
Did one better and expanded this into it's own paragraph to give both options and explain why.